torchbenchmark/models/torchrec_dlrm/requirements.txt

```diff
@@ -0,0 +1,2 @@
+torchrec-nightly
```

Comment:
this will be using nightly of TorchRec/ FBGEMM GPU? is that typical for torchbench?

Is it also using nightly torch?

Reply:
Yes, all torchbench CI workflows are using nightly pytorch by default. My concern of using URL to install is that users may experience longer time to compile and install the packages.